ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -619,4 +623,15 @@ static void maybeCreateProcessingLogStream(
 
     commandQueue.enqueueCommand(configured.get());
   }
+
+  private static void maybeConfigureRocksDBConfigSetter(final KsqlConfig ksqlConfig) {
+    final Map<String, Object> streamsProps = ksqlConfig.getKsqlStreamConfigProps();
+    final Class<?> clazz =
```

Comment:
Is this always a `Class` or can it be a `String` containing the class name?  I _think_ its always a class... have you spun up an actual instance with the config set in the props file? That would confirm it...